File: app/src/main/java/com/wealoha/social/beans/common/CountService.java
package com.wealoha.social.beans.common;

import retrofit.Callback;
import retrofit.http.GET;

import com.wealoha.social.beans.Result;
import com.wealoha.social.impl.ServerUrlImpl;

/**
 * 
 * 
 * @author javamonk
 * @see
 * @since
 * @date 2014-12-12 上午10:29:05
 */
public interface CountService {

	/**
	 * 计数
	 * 
	 * @param callback
	 */
	@GET(ServerUrlImpl.COUNT)
	public void count(Callback<Result<CountData>> callback);
}

File: app/src/main/java/com/wealoha/social/api/CountService.kt
package com.wealoha.social.api

import com.wealoha.social.beans.Result
import com.wealoha.social.beans.common.CountData
import com.wealoha.social.impl.ServerUrlImpl
import retrofit.Callback
import retrofit.http.GET

/**
 *
 *
 * @author javamonk
 * @see
 * @since
 * @date 2014-12-12 上午10:29:05
 */
interface CountService {
    /**
     * 计数
     *
     * @param callback
     */
    @GET(ServerUrlImpl.COUNT)
    fun count(callback: Callback<Result<CountData?>?>?)
}